### ISSUE TITLE
Direct Hit and Direct Crit Decimal %

### ISSUE THREAD

**Issue by KillianRyan** [ravahn/FFXIV_ACT_Plugin#86]
Hey Ravahn,

When using the output text formatting options for direct hit and direct critical hit percentages, they display as xx.x% while all other percentages like critical hit and damage done are non decimal xx%. I was hoping the former could be changed to match the latter so they are all xx% and non decimal. Here is an example of what it's like currently: http://imgur.com/a/yNIIa

Thanks!

**Comment by ravahn** [comment]
I'll take a look.

**Comment by christianallred** [comment]
I think that xx.xx% would actually be the most helpful. I get that it is a bit longer, but i like the precision. Just my opinion.

**Comment by ravahn** [comment]
I decided to update this to remove the decimal place for the export variable to be consistent with the built-in ACT export variables for overlays.  The in-game columns for direct hit percent/DH crit % still have a decimal place.  This is now available in v1.5.1.4.

**Comment by KillianRyan** [comment]
Hey Ravahn, thanks for the update. After testing it out, I confirmed that the output is somehow broken (no display) when critical direct hit % is an export variable. But direct hit % is working fully. I also saw a reddit thread that made mention of this update breaking their rainbow mage but I think that has more to do with rainbow mage needing to accommodate to your new export format but thought I would include it just in case. https://www.reddit.com/r/ffxiv/comments/6jzeib/is_the_new_ffxiv_act_update_1514_causing_any/
